katex.js

```diff
@@ -47,6 +50,9 @@ if (typeof document !== "undefined") {
  * Parse and build an expression, and return the markup for that.
  */
 var renderToString = function(expression, options) {
+    if (!(typeof expression === 'string' || expression instanceof String)) {
+        throw new TypeError('renderToString requires string typed expression');
+    }
     var settings = new Settings(options);
 
     var tree = parseTree(expression, settings);
```

Comment:
Implemented in https://github.com/Khan/KaTeX/pull/540/commits/73b9fa5cc69fa586198ac6f84f6772a1d237036c